From stable-bounces@linux.kernel.org Fri Dec  2 12:55:14 2005
Message-ID: <4390B3A1.9040801@gentoo.org>
Date: Fri, 02 Dec 2005 20:50:41 +0000
From: Daniel Drake <dsd@gentoo.org>
To: stable@kernel.org
Cc: r3pek@gentoo.org
Subject: DVB: BUDGET CI card depends on STV0297 demodulator

From: Carlos Silva <r3pek@gentoo.org>

BUDGET_CI card depends on STV0297 demodulator.
This patch solves a DVB driver compile error introduced in 2.6.14

Signed-off-by: Johannes Stezenbach <js@linuxtv.org>
Signed-off-by: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Linus Torvalds <torvalds@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/media/dvb/ttpci/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.14.3.orig/drivers/media/dvb/ttpci/Kconfig
+++ linux-2.6.14.3/drivers/media/dvb/ttpci/Kconfig
@@ -81,6 +81,7 @@ config DVB_BUDGET_CI
 	tristate "Budget cards with onboard CI connector"
 	depends on DVB_CORE && PCI
 	select VIDEO_SAA7146
+	select DVB_STV0297
 	select DVB_STV0299
 	select DVB_TDA1004X
 	help
